pages/11ty/url.filter.js

```diff
@@ -0,0 +1,5 @@
+const ghurl = (url) => 'https://github.com/exadel-inc/esl/' + url;
```

Comment:
Use `site.config.js` to get this information from `site.yml` but not hadcode and duplicate it in code